### ISSUE TITLE
vxm does not return 

### ISSUE THREAD

**Issue by srki** [DrTimothyAldenDavis/GraphBLAS#13]
### Unexpected behavior
When GrB_vxm is called with empty w, u, A, and non-empty mask, GrB_vxm never returns.

### Example code

``` c
#include "GraphBLAS.h"

int main(int argc, char *argv[]) {
    GrB_init(GrB_BLOCKING);

    size_t size = 100;

    GrB_Vector w;
    GrB_Vector_new(&w, GrB_BOOL, size);

    GrB_Vector u;
    GrB_Vector_new(&u, GrB_BOOL, size);

    GrB_Vector mask;
    GrB_Vector_new(&mask, GrB_BOOL, size);
    for (int i = 0; i < 5; i++) {
        GrB_Vector_setElement_BOOL(mask, true, i);
    }

    GrB_Matrix A;
    GrB_Matrix_new(&A, GrB_BOOL, size, size);

    GrB_vxm(w, mask, GrB_NULL, GxB_LOR_LAND_BOOL, u, A, GrB_NULL);

    GrB_finalize();
}
```

### Detailed Description
 
If the inputs _u_ and _A_ are empty, the hash table used in loop [GB_AxB_saxpy3_symbolic.c#L173](https://github.com/DrTimothyAldenDavis/GraphBLAS/blob/2f98bdf1dc24f81e49505bcf27c93e2a8bb1641e/Source/GB_AxB_saxpy3_symbolic.c#L173) will have space for 4 elements.
If the _mask_ has more than 4 elements, loop [GB_AxB_saxpy3_symbolic.c#L167](https://github.com/DrTimothyAldenDavis/GraphBLAS/blob/2f98bdf1dc24f81e49505bcf27c93e2a8bb1641e/Source/GB_AxB_saxpy3_symbolic.c#L167) will not be able to find space for all elements, and it will loop forever.

**Comment by DrTimothyAldenDavis** [comment]
Thanks for the bug report.  I've replicated the problem and I see why it's happening.  It shouldn't take too long to fix.

**Comment by DrTimothyAldenDavis** [comment]
Fixed with GraphBLAS v3.3.3.  Thanks for finding the bug and the very helpful isolation.

**Comment by srki** [comment]
Thank you for the very fast bug fix. I ran a few more tests, and everything runs as expected. Thank you!
